### DISCUSSION TITLE
How do I remove pool from a program chia

### DISCUSSION THREAD

**Discussion by upanime** [Chia-Network/chia-blockchain#7251] [Support]
Hello
How do I remove pool from a program chia
![image](https://user-images.githubusercontent.com/5046409/124980484-1b43ff00-e03d-11eb-9b04-e90b4970f6c4.png)
Thank you

**Comment by ZielReaver** [comment]
from what I have read so far you can not. I did see some place someone talking second hand about someone else went into the code and manually deleted them but if you resynced they would return but so far have not found out were that was posted and how.

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
